debian/postinst

```diff
@@ -29,8 +29,17 @@ insert_nss_entry() {
 }
 
 action="$1"
+previous_version="$2"
 
 if [ configure = "$action" ]; then
     pam-auth-update --package
     insert_nss_entry
+
+    # We installed /etc/authd with permissions 777 - umask in version 0.3.7 and earlier
+    if dpkg --compare-versions "$previous_version" le 0.3.7; then
```

Comment:
Fine, as matter of exercise (not for now, just for future packaging...) normally we do:

`dpkg --compare-version "$previous_version" lt-nl 0.3.7~` so that anything that is previous to the current (excluding the pre-releases that use `~`, should be fixed).

As ideally we don't need to fix `0.3.7` (since it's not released yet).

Reply:
> dpkg --compare-version "$previous_version" lt-nl 0.3.7~ so that anything that is previous to the current (excluding the pre-releases that use ~, should be fixed).

I see

> As ideally we don't need to fix 0.3.7 (since it's not released yet).

It is, 0.3.7 is the last release. The new one will be 0.4.0.

Reply:
Yeah, sorry I meant `4.0.0`.